src/validator/GoValidator.cpp

```diff
@@ -949,8 +949,9 @@ Status GoValidator::buildColumns() {
     }
 
     auto pool = qctx_->objPool();
-    if (!exprProps_.srcTagProps().empty() || !exprProps_.edgeProps().empty() ||
-        !exprProps_.dstTagProps().empty()) {
+    if (!exprProps_.inputProps().empty() || !exprProps_.varProps().empty() ||
+        !exprProps_.srcTagProps().empty() || !exprProps_.edgeProps().empty() ||
+        !exprProps_.dstTagProps().empty() || from_.fromType != FromType::kInstantExpr) {
```

Comment:
Extract these conditions to a function with a meaningful name will be better, like `isPropsEmpty()` or something ?